### PR TITLE
FIX-FAT-999: fixed an issue on the scenario of lean features 

### PR DESCRIPTION
**Pull request by yusuf-murodov-epam**

### CODE REVIEW

File: mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature [hunk 1]
```diff
@@ -707,7 +708,7 @@ Feature: Loans tests
 
     # post owner, manual charge and payment method
     * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostOwner')
-    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostManualCharge') { extManualChargeId: #(extManualChargeId) }
```
Comment (viktor-draban) [inline mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:711]: I investigated error logs more deeply and found out the truly reason of fail, so besides changes you have done please replace couple of lines
image
into 
image

Comment (yusuf-murodov-epam) [reply mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:711]: fixed

Comment (viktor-draban) [reply mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:711]: please replace "And" after test into asterix "*" you can see what I mean on screenshot

Comment (yusuf-murodov-epam) [reply mod-circulation/src/main/resources/vega/mod-circulation/features/loans.feature:711]: fixed

**Review by viktor-draban — APPROVED** [review]

**Review by sherzodn — APPROVED** [review]

**Pull request merged**